packages/manager/apps/adp/adp.less

```diff
@@ -0,0 +1,73 @@
+@import '~flag-icon-css/less/flag-icon.less';
```

Comment:
Is there no other solution than define custom styles in the apps. If we do that, `adp` module won't be standalone (and require to define style in each application using it)

Reply:
There are two solutions. Define these classes in app or module. These classes are already defined in ovh-manager-cloud. If we add them to app, it will be duplicate data. For the standalone module to work, we have added them to app. When deployed as standalone, app classes are used and when deployed in manager, classes in ovh-manager-cloud are used. Another way is to isolate these into separate module and use it in ADP, PCI etc

Reply:
Is `cloud-styles` module may help you to address this?